webpack.config.js

```diff
@@ -278,5 +278,7 @@ function getDefaults(grunt) {
             'pt_BR',
             'pl',
         ],
+
+        userOnlineMinutes: 15,
```

Comment:
What about checking websocket status on the server instead? If the websocket is connected - it would mean the user is online.

Reply:
if you keep browser tab opened you would still appear to be online, while you might not be doing anything for hours

Reply:
Sounds accurate to me. "online" means connected, no?

Reply:
yep that would mean browser/device is online, but doesn't say much about the user, so not sure how useful that would be actually.

Reply:
It depends what the use case is. In most software "online" indicator is so you know if the user is reachable and can respond. I have the slack tab open and have not done anything there for hours, but I am reachable. Is the use case different here?

Reply:
lets ask @karelpetrak I don't see the original requirements anywhere, not sure if the slack usecase works well for newsroom

Reply:
Some applications use 3 states to determine a users' online status:
1. offline
2. online/active
3. online/inactive

IMO, this `userOnlineMinutes` could also be used on the back-end at some point. Would it be beneficial to have this setting in `settings.py` and exposed to the front-end via the `app.client_config` attribute?

Reply:
I wasn't sure what would be the usecase on the server. It would make sense to let user set their status maybe, now we're tracking last activity time, but that would be something for next release